int64 TurtleNum
int64 TurtleGoal